src/test/resources/test.conf

```diff
@@ -182,6 +182,13 @@ storage {
             reply-secondary {
                 engine = "s3"
             }
+            blob-files {}
+        }
+    }
+    layer2.spaces {
+        blob-files {
+                readPermission = "disabled"
+                description = "blob-files {}"
```

Comment:
just for the record - the description isn't necessary in test configs :)

Reply:
done